ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,996 @@
+# RZG2L SBC board #
```

Comment:
Update this section to show that we are porting and creating a custom Ubuntu-core image for RZG2L-SBC

Reply:
The first line "RZG2L SBC board" should change to "Ubuntu Core System development on RZG2L SBC board"